packages/docs/pages/GettingStarted/GettingStartedPage.tsx

```diff
@@ -48,6 +48,19 @@ export default () => {
               Beta Community Group
             </Link>
           </List.Item>
+          <List.Item>
+            <Link
+              href="https://www.figma.com/file/jTVuUkiZ1j3rux8WHG4IKK/BigDesign-UI-Kit?node-id=0%3A1/duplicate"
```

Comment:
May be a better idea to change url to `https://www.figma.com/file/jTVuUkiZ1j3rux8WHG4IKK/BigDesign-UI-Kit`. In fact there's no need to include the title, but I'm not sure if it would still redirect to the file if the title were to change (but probably would).

Reply:
Updated to remove that, as you were correct.